enterprise/server/ociregistry/ociregistry.go

```diff
@@ -353,9 +360,17 @@ func (r *registry) handleBlobsOrManifestsRequest(ctx context.Context, w http.Res
 	}
 }
 
+func trackCache(cacheType, eventType string) {
```

Comment:
nit: the name `trackCache` is a little too vague, it doesn't tell you what the function does. I would call it something like `updateEventMetric`.